web/app/types/project.d.ts

```diff
@@ -15,14 +15,15 @@ export interface JiraObject {
 }
 
 export interface HermesProject {
-  id: string; // at least in Mirage...
+  id: string;
   title: string;
   status: ProjectStatus;
   hermesDocuments?: RelatedHermesDocument[];
   description?: string;
-  jiraObject?: JiraObject;
+  jiraIssueID?: string;
+  jiraIssue?: JiraIssue;
```

Comment:
Is `jiraIssueID` supposed to represent the internal Jira ID (if so, curious how we use that)? Can we put that as part of a `JiraIssue`? Note/caveat: I haven't dove into the Jira part of the backend work yet so I'm cool with this changing as we start to incorporate that.

Reply:
I interpreted JiraIssueID to be an _external_ reference, like "LABS-123" (as in `test-org.atlassian.com/issues/LABS-123` that would be used to query the Jira API for the full object.